pkg/platform/abstract/platform.go

```diff
@@ -492,9 +510,14 @@ func (ap *Platform) prettifyProcessorLogLine(log []byte) (string, string, error)
 		logStruct.Time = &unparsedTime
 		logStruct.Level = wrapperLogStruct.Level
 		logStruct.Message = wrapperLogStruct.Message
+		logStruct.Name = wrapperLogStruct.Name
 
-		more := common.CreateKeyValuePairs(wrapperLogStruct.With)
-		logStruct.More = &more
+		if wrapperLogStruct.With != nil {
+			workerID = wrapperLogStruct.With["worker_id"]
```

Comment:
Move this inside `inferWorkerID()` so the logic is not spread out

Reply:
I must do it in that part because afterwards I cannot access `wrapperLogStruct.With` because the sdk's logger is being normalized to match the generic `logStruct` (defined earlier).
And `inferWorkerID()` is being called only if the worker_id wasn't explicitly given as an arg as commented in the code